bigbluebutton-html5/imports/ui/components/audio/audio-controls/styles.scss

```diff
@@ -31,8 +31,32 @@
   }
 
   &.glow {
-    span:first-child{
-      box-shadow: 0 0 0 1px #ffffff, 0 0 0 1px;
-    }
+    -webkit-animation: pulse 1s infinite ease-in-out;
+    animation: pulse 1s infinite ease-in-out;
```

Comment:
Is not it better to use ease-in? with ease-in-out it seems that the user has stopped talking.